Docs for Executor (#2467)
Qiskit/qiskit-ibm-runtime: docs/guides/executor_basic.rst

The Executor: A quick-start guide
================================= 

This guide provides a basic overview of the :class:`~.Executor`, a runtime program that allows
executing :class:`~.QuantumProgram`\s on IBM backend. At the end of this guide, you will
know how to:

> * Initialize a :class:`~.QuantumProgram` to specify one or more circuit tasks.

Reviewer: ```suggestion
* Initialize a :class:`~.QuantumProgram` with your workload.
```

* Run :class:`~.QuantumProgram`\s on IBM backends using the :class:`~.Executor`.
* Interpret the outputs of the :class:`~.Executor`.

> In the reminder of the guide, we consider a circuit that generates a three-qubit GHZ state, rotates

Reviewer: ```suggestion
In the remainder of the guide, we consider a circuit that generates a three-qubit GHZ state, rotates
```

.. code-block:: python

    from qiskit_ibm_runtime import QiskitRuntimeService, Executor

    service = QiskitRuntimeService()
    backend = service.least_busy(operational=True, simulator=False)

We can now begin by taking a look at the inputs to the :class:`~.Executor`, the :class:`~.QuantumProgram`\s.

The inputs to the Executor: Quantum Programs
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

A :class:`~.QuantumProgram` is an iterable of
:class:`~.qiskit_ibm_runtime.quantum_program.QuantumProgramItem`\s. Each of these items represents a
different circuit task for the :class:`~.Executor` to perform. Typically, they own:

* a :class:`~qiskit.circuit.QuantumCircuit` with static, non-parametrized gates;
* or a parametrized :class:`~qiskit.circuit.QuantumCircuit`, together with an array of parameter values;
* or a parametrized :class:`~qiskit.circuit.QuantumCircuit`, together with a
  :class:`~samplomatic.samplex.Samplex` to generate randomize arrays of parameter values.

Let us take a closer look at each of these items and how to add them to a :class:`~.QuantumProgram`\.

In the cell below, we initialize a :class:`~.QuantumProgram` and specify that we wish to perform ``1024``

> shots per item in the program. Next, we append a version of our target circuit with set parameters,

Reviewer: the current wording makes it sounds like 1024 is the total shot budget for  each item

```suggestion
shots for every configuration of each item in the program. Next, we append a version of our target circuit with set parameters,
```
